File: frontend/src/types/supplier.ts
// טיפוסי הדומיין של ספק — משקפים את מודל ה-Supplier ב-Backend
export interface Supplier {
  id: number
  name: string
  contactPerson: string | null
  phone: string | null
  isActive: boolean
}

// מה ששולחים ליצירה/עדכון — בלי ה-Id שנוצר בשרת
export type SupplierInput = Omit<Supplier, 'id'>

File: backend/Models/Supplier.cs
namespace Backend.Models;

public class Supplier
{
   public int Id { get; set; }
   public string Name { get; set; } = string.Empty;
   public string? ContactPerson { get; set; }
   public string? Phone { get; set; }
   public bool IsActive { get; set; } = true;
}
